### ISSUE TITLE
GetLatestVersionNumber() not working in Android version of 1.1.1

### ISSUE THREAD

**Issue by betulerdim** [edsnider/latestversionplugin#10]
## Bug

Plugin Version:  1.1.1
Platform (e.g., Android v21):  

Description:    I installed version of 1.1.1 But GetLatestVersionNumber()  not working on this version. I updated regex to "<div[^>]*>Current Version</div><span[^>]*><div[^>]*><span[^>]*>(.*?)<" now it is working but in nuget not working. Please update nuget

**Comment by elyfran** [comment]
same problem here.

**Comment by nmano2188** [comment]
<div[^>]*>Current Version</div><span[^>]*><div[^>]*><span[^>]*>(.*?)<"

**Comment by edsnider** [comment]
Fix is available in beta package on NuGet: https://www.nuget.org/packages/Xam.Plugin.LatestVersion/1.1.2-beta65

**Comment by elyfran** [comment]
> Fix is available in beta package on NuGet: https://www.nuget.org/packages/Xam.Plugin.LatestVersion/1.1.2-beta65

Thank you

**Comment by KeesAlderliesten** [comment]
> ```
> <div[^>]*>Current Version</div><span[^>]*><div[^>]*><span[^>]*>(.*?)<
> ```

Got the same problem and didn't realize there was a beta already. So I tested the regex on Regex101.com and that gave this error: / An unescaped delimiter must be escaped with a backslash (\)
So I modified the regex to this <div[^>]*>Current Version<\\/div><span[^>]*><div[^>]*><span[^>]*>(.*?)<  (double backslash because c#) and then it worked also.

**Comment by edsnider** [comment]
> Got the same problem and didn't realize there was a beta already. So I tested the regex on Regex101.com and that gave this error: / An unescaped delimiter must be escaped with a backslash ()
> So I modified the regex to this <div[^>]_>Current Version<\/div><span[^>]_><div[^>]_><span[^>]_>(.*?)< (double backslash because c#) and then it worked also.

Correct, it needs to be escaped on Regex101 - but not in the code.  Can you confirm by testing the beta build?

**Comment by KeesAlderliesten** [comment]
Hi Ed,

The beta works fine!  (but with the backslash in code it worked also..)

But I go for your beta, easier to update.

Thanks!

Kind regards,

Kees Alderliesten


Van: Ed Snider <notifications@github.com>
Verzonden: vrijdag 21 december 2018 15:36
Aan: edsnider/latestversionplugin <latestversionplugin@noreply.github.com>
CC: Kees Alderliesten <kees@frames.nl>; Comment <comment@noreply.github.com>
Onderwerp: Re: [edsnider/latestversionplugin] GetLatestVersionNumber() not working in Android version of 1.1.1 (#10)


Got the same problem and didn't realize there was a beta already. So I tested the regex on Regex101.com and that gave this error: / An unescaped delimiter must be escaped with a backslash ()
So I modified the regex to this <div[^>]>Current Version</div><span[^>]><div[^>]><span[^>]>(.*?)< (double backslash because c#) and then it worked also.

Correct, it needs to be escaped on Regex101 - but not in the code. Can you confirm by testing the beta build?
